soc/nxp/imx/imx8m/m4_mini/linker.ld

```diff
@@ -9,7 +9,7 @@
 SECTIONS
      {
```

Comment:
If there is no much of a trouble would be nice to have an explanation in the commit message of WHY this is happening.(not a blocker though).

Other than that this should be rebased on top of  hwmv2 changes.

Reply:
I have rebased it already? And the commit message states the reason? Do you wish to see a more elaborate explanation?

Reply:
@pdgendt 

> could result in linux kernel panics if this change isn't included.

Is the effect not the cause. I would have been more curious about the cause of the kernel crash. But this might be bikeshedding on my part, so you can skip this comment :)

Reply:
I've added some more context to the commit message, I hope this makes is clear, thanks!